### PR TITLE
doc: tm_strcontains documentation.

### PR DESCRIPTION
**Pull request by i4ki**

### CODE REVIEW

File: cli/reference/functions/tm_strcontains.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: tm_strcontains | Functions | Configuration Language
+description: The tm_strcontains function checks whether a substring is within another string.
+---
+
+# `tm_strcontains` Function
+
+`tm_strcontains` function checks whether a substring is within another string.
```
Comment (AnnuCode) [inline cli/reference/functions/tm_strcontains.md:8]:
```suggestion
The `tm_strcontains` function checks if a given string contains a specific substring.
```

File: cli/reference/functions/tm_strcontains.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+tm_strcontains("hello world", "wor")
+true
+
+tm_strcontains("hello world", "wod")
+false
+```
+
+## Related Functions
+
+* [`tm_startswith`](./tm_startswith.md) takes two values: a string to check and a prefix string. The function returns true if the string begins with that exact prefix.
```
Comment (AnnuCode) [inline cli/reference/functions/tm_strcontains.md:26]:
```suggestion
* [`tm_startswith`](./tm_startswith.md) checks if a string starts with a specified prefix. It takes two arguments: the string and the prefix, returning true if the string starts with the exact prefix.
```

File: cli/reference/functions/tm_strcontains.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+* [`tm_endswith`](./tm_endswith.md) takes two values: a string to check and a suffix string. The function returns true if the first string ends with that exact suffix.
```
Comment (AnnuCode) [inline cli/reference/functions/tm_strcontains.md:27]:
```suggestion
* [`tm_endswith`](./tm_endswith.md) checks if a string ends with a specified suffix. It takes two arguments: the string and the suffix, returning true if the string ends with the exact suffix.
```

**Review by soerenmartius — DISMISSED** [review]
LGTM, @i4ki this needs to be added to the sidebar also

**Comment by i4ki** [comment]
it was added to the sidebar.

**Review by AnnuCode — APPROVED** [review]

**Pull request merged**